### DISCUSSION TITLE
A question about ibip method

### DISCUSSION THREAD

**Discussion by amisatoayak** [PMEAL/porespy#853] [Q&A]
Dear professor

Recently i am trying to use ibip function to calculate air invading in water. But I did not find the parameters similar to 'delta_rho' in drainage method.
 I wonder whether the density of fluids need to be concerned in this algorithm? 
Or can this algorithm be used as long as it is between two insoluble phases like air and water?
Thank you in advance.

**Comment by jgostick** [comment]
We do not yet have the ibip method working with gravity, sorry.
